db/schema.rb

```diff
@@ -322,6 +322,15 @@
     t.index ["job_id"], name: "index_form526_job_statuses_on_job_id", unique: true
   end
 
+  create_table "form526_opt_ins", id: :serial, force: :cascade do |t|
```

Comment:
This change appears every time a `make rebuild` is done

Reply:
It appears that your branch doesn't have this change: https://github.com/department-of-veterans-affairs/vets-api/commit/42b7769ffba861692eb5bdabfceb0b148c2bb52c#diff-cbf8b25d6d853acf58fa9057841f407d29ffe90649c75cf9e3f51b2d6e3aa1d3

Reply:
i haven't done a `make migrate` in a while which is why was continuing to show up when doing a `make rebuild`

Reply:
I removed this change but even after a `make migrate` > `make rebuild` the change still shows up